wrappers/python/nntile/layer/layer_norm.py

```diff
@@ -339,3 +341,37 @@ def backward_async(self):
         self.tmp_y_value.invalidate_submit()
         # dX can offloade from GPU
         self.x.grad.wont_use()
+
+    @classmethod
+    def from_torch(cls,
+        torch_layer, x: TensorMoments,
+        axis: int, eps: float,
+        next_tag: int, redux: bool = False
+    ):
+        nntile_layer, next_tag = cls.generate_simple(x, axis, eps**2, next_tag, redux)
```

Comment:
`eps` must be read from `torch_layer`. Be careful with `eps`, as it is square rooted at line 74. `axis` shall also be deduced automatically without passing it through parameters of `from_torch` method. We assume `LayerNorm` takes as input `(n_batch, n_seq, n_emb)` tensor and, therefore, `axis` shall be `0`.